client/jetpack-cloud/sections/partner-portal/primary/payment-method-list/index.tsx

```diff
@@ -0,0 +1,123 @@
+/**
+ * External dependencies
+ */
+import React, { ReactElement } from 'react';
+import { useSelector } from 'react-redux';
+import { useTranslate } from 'i18n-calypso';
+import page from 'page';
+
+/**
+ * Internal dependencies
+ */
+import Main from 'calypso/components/main';
+import CardHeading from 'calypso/components/card-heading';
+import DocumentHead from 'calypso/components/data/document-head';
+import SidebarNavigation from 'calypso/jetpack-cloud/sections/partner-portal/sidebar-navigation';
+import { Card } from '@automattic/components';
+import { getAllStoredCards } from 'calypso/state/stored-cards/selectors';
+import QueryStoredCards from 'calypso/components/data/query-stored-cards';
+import {
+	VisaLogo,
+	MastercardLogo,
+	AmexLogo,
+} from 'calypso/my-sites/checkout/composite-checkout/components/payment-logos';
+import EllipsisMenu from 'calypso/components/ellipsis-menu';
+import PopoverMenuItem from 'calypso/components/popover/menu-item';
+import Gridicon from 'calypso/components/gridicon';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+function StoredCard( props ): ReactElement {
+	let logo;
+	const translate = useTranslate();
+
+	if ( 'visa' === props.card.card_type ) {
+		logo = <VisaLogo />;
+	} else if ( 'mastercard' === props.card.card_type ) {
+		logo = <MastercardLogo />;
+	} else if ( 'amex' === props.card.card_type ) {
+		logo = <AmexLogo />;
+	}
+
+	const expiry = new Date( props.card.expiry );
+	const month = expiry.getMonth();
+	const year = expiry.getFullYear().toString().substr( 2, 2 );
+
+	return (
+		<div className="payment-method__stored-card">
+			<div className="payment-method__stored-card-header">
+				<div className="payment-method__stored-card-header-left">{ logo }</div>
+
+				<div className="payment-method__stored-card-header-right">
+					<Gridicon icon="checkmark" size={ 18 } />
+					<div className="payment-method__stored-card-primary">
+						<span className="payment-method__stored-card-primary-text">
+							{ translate( 'Primary' ) }
+						</span>
+					</div>
+					<EllipsisMenu className="payment-method__stored-card-ellipsis" position="bottom">
+						<PopoverMenuItem>{ translate( 'Make primary' ) }</PopoverMenuItem>
+						<PopoverMenuItem>{ translate( 'Update details' ) }</PopoverMenuItem>
+						<PopoverMenuItem>{ translate( 'Delete card' ) }</PopoverMenuItem>
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **23** times:
`translate( 'Delete Card', { context: 'Button label'} )` *ES Score: 10*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/jetpack-partner-payment)